systest/scripts/install_lbaasv2.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -ex
+
+GLOBAL_ROUTED_MODE=False
+OS_CONTROLLER_IP=`/tools/bin/tlc --sid ${TEST_SESSION} symbols \
+    | grep openstack_controller1ip_data_direct \
+    | awk '{print $3}' | xargs`
+SSH_CMD="ssh -i /home/jenkins/.ssh/id_rsa_testlab -o StrictHostKeyChecking=no testlab@${OS_CONTROLLER_IP}"
+BIGIP_IP=`${SSH_CMD} "cat /home/testlab/ve_mgmt_ip"`
+BIGIP_IP=${BIGIP_IP%%[[:cntrl:]]}
+NETWORK_TYPES=${TEST_TENANT_NETWORK_TYPE}
+AGENT_LOC=git+https://github.com/F5Networks/f5-openstack-agent.git@${TEST_OPENSTACK_DISTRO}
+DRIVER_LOC=git+https://github.com/F5Networks/f5-openstack-lbaasv2-driver.git@${TEST_OPENSTACK_DISTRO}
+NEUTRON_DRIVER_LOC=https://raw.githubusercontent.com/F5Networks/neutron-lbaas/stable/${TEST_OPENSTACK_DISTRO}/neutron_lbaas/drivers/f5/driver_v2.py
```

Comment:
The TEST_OPENSTACK_DISTRO is appropriate here however, since stable is built in. We could use branch, then allow the substitution in the url here replace 'stable/<os_release>'.